modules/ROOT/pages/bat-top.adoc

```diff
@@ -2,34 +2,28 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: bat-workflow-test.adoc
 
-You can use API functional monitoring to run a test or suite of tests on an API. API functional monitoring is based on Blackbox Automated Testing (BAT).
+You can use API Functional Monitoring to run a test or suite of tests on an API. API Functional Monitoring is based on Blackbox Automated Testing (BAT).
 
-Using API functional monitoring, you can automate testing and get test reports. You can test before pushing code into your repository and before deploying to production. Tests can verify API functionality and validate results the API generates. You can monitor APIs at each stage of the software development life cycle and in production.
+Using API Functional Monitoring, you can automate testing and get test reports. You can test before pushing code into your repository and before deploying to production. Tests can verify API functionality and validate results the API generates. You can monitor APIs at each stage of the software development life cycle and in production.
 
-The prerequisites for performing API functional testing and scheduling are:
+The prerequisites for testing and scheduling are:
 
-* Deployed API
-+
-You can deploy an API on Anypoint Platform or your own server.
+* One or more deployed APIs
 * Installation of BAT.
-* A YAML configuration file (manifest file).
-* At least one test file.
+* A directory that contains a test suite.
+* At least one test file in that directory.
 +
-** You write files in Behavior Driven Development (BDD) syntax using a code or text editor, such as IntelliJ.
-** You save the files using the .dwl extension for processing by DataWeave.
+You write tests in Behavior Driven Development (BDD) syntax using a code or text editor, such as IntelliJ. Test files must use the `.dwl` extension for processing by DataWeave.
 
 Additional prerequisites for monitoring scheduled tests in the cloud are:
 
 * An account on Anypoint Platform.
 * A reporting tool supported by BAT.
```

Comment:
Not A prerequisite. You can schedules without any reporting tool.

Reply:
Resolved.